sdk/apimanagement/api-management-custom-widgets-scaffolder/CHANGELOG.md

```diff
@@ -1,15 +1,15 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2022-08-22)
```

Comment:
Our monthly release is scheduled on 10/06, is it okay if we release then?

Reply:
One and half month is too late for us. As there is a bug in the package, we'll need to release it before we make it GA. I've scheduled a meeting with you to sync on the release process. Thank you!